reference/orders.v3.yml

```diff
@@ -2337,6 +2354,26 @@ definitions:
         type: string
         minLength: 0
         maxLength: 1000
+  MerchantOverride:
+    type: object
+    title: Merchant Calculated Override
+    description: |
+      Merchant explicitly provided override based on their own calculation.
+
+      This gives merchants the flexibility to
+      - bypass any tax correction due to tax rate/providers changes between when the order is placed and refund is initiated
+      - use explicit values calculated by external systems (eg. merchants' own EPR or OMS)
+
+      NB. When using the override, BC internal tax based refund calculation is skipped and therefore order/taxes records are not updated.
```

Comment:
NB.?

When using the override, BC internal tax-based refund calculation is skipped, and therefore, order/taxes records are not updated.

Reply:
@bc-vincent-zhao could you define NB or is this a typo?

Reply:
sorry, I think I meant  `N.B.` (An [abbreviation](https://www.dictionary.com/browse/abbreviation) for the [Latin](https://www.dictionary.com/browse/latin) phrase nota bene, meaning “note well.” It is used to emphasize an important point.) ?

If that's not the standard, feel free to remove it.

Reply:
Gotcha! Thanks for the clarification. We'll get this up shortly.